Scala/Scala.sublime-syntax

```diff
@@ -147,6 +147,46 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '(@){{plainid}}\.'
```

Comment:
How about specializing this to `punctuation.accesor.dot.scala`, like it's done for comment punctuation?

I suggested this a few months ago already for C# iirc.